25_pair_code_functions.h

```diff
@@ -0,0 +1,31 @@
+//File contains enums, structure declarations and function protoypes to implement 25_pair_code_functions
+#ifndef DEFINE_MAIN_ONCE
+#define DEFINE_MAIN_ONCE
+
+#include <stdio.h>
+#include <assert.h>
+
+enum MajorColor {WHITE, RED, BLACK, YELLOW, VIOLET};
+enum MinorColor {BLUE, ORANGE, GREEN, BROWN, SLATE};
+
+typedef struct {
+    enum MajorColor majorColor;
+    enum MinorColor minorColor;
+} ColorPair;
+
+ColorPair GetColorFromPairNumber(int pairNumber);
+
+void testNumberToPair(int pairNumber,
```

Comment:
Test code is part of production code. Production code needs to be kept separately from test code so that test code can be excluded during shipment of software to field.
Separation helps in maintenance and most importantly helps in testing production code independently and effectively.